copy-example-data.sh

```diff
@@ -5,5 +5,3 @@ mkdir -p examples-build/data
 cp -r examples/data/photos examples-build/data/
 cp -r examples/data/photos.json examples-build/data/
 cp -r examples/data/rao-2014-gm12878-chr-22-peaks.json examples-build/data/
-cp -r examples/data/apple.ndjson examples-build/data/
```

Comment:
I don't think these deletions are expected. Somehow this branch got out of sync with the master. I merged the master into this branch but it doesn't solve the issue. Can you fix these issues manually?